src/game/server/gamecontext.cpp

```diff
@@ -1521,6 +1521,9 @@ void CGameContext::OnInit()
 		g_Config.m_SvPlayerSlots = Server()->MaxClients();
 
 #ifdef CONF_DEBUG
+	// clamp dbg_dummies to 0..MaxClients-1
+	if(Server()->MaxClients() <= g_Config.m_DbgDummies)
+		g_Config.m_DbgDummies = Server()->MaxClients()-1;
```

Comment:
I think we could allow `g_Config.m_DbgDummies = Server()->MaxClients()` since it drops them anyway